### PR TITLE
sync yaml py Dockerfile json with /en

### PR DESCRIPTION
**Pull request by tanjunchen**
/cc @tengqm 
/cc @SataQiu 

Please see this pr. Thank you very much . 
see https://github.com/kubernetes/website/pull/19881

### CODE REVIEW

File: content/zh/examples/admin/resource/quota-mem-cpu-pod-2.yaml [hunk 1]
```diff
@@ -9,8 +9,8 @@ spec:
     resources:
       limits:
         memory: "1Gi"
-        cpu: "800m"
```
Comment (tanjunchen) [inline content/zh/examples/admin/resource/quota-mem-cpu-pod-2.yaml]: en change is not good . so i will open pr to fix it .

File: content/zh/examples/admin/sched/my-scheduler.yaml [hunk 2]
```diff
@@ -1,67 +1,67 @@
...
-            port: 10251
-        resources:
-          requests:
-            cpu: '0.1'
-        securityContext:
-          privileged: false
-        volumeMounts: []
-      hostNetwork: false
-      hostPID: false
-      volumes: []
```
Comment (tanjunchen) [inline content/zh/examples/admin/sched/my-scheduler.yaml]: the content of en/ and zh is the same, this script detects the  difference . maybe the reason of os.

File: content/zh/examples/service/networking/ingress.yaml [hunk 3]
```diff
@@ -1,9 +1,9 @@
-apiVersion: networking.k8s.io/v1beta1
-kind: Ingress
-metadata:
-  name: test-ingress
-spec:
-  backend:
-    serviceName: testsvc
-    servicePort: 80
```
Comment (tanjunchen) [inline content/zh/examples/service/networking/ingress.yaml]: ditto

File: content/zh/examples/admin/sched/my-scheduler.yaml (hunk 2 above)
Comment (tanjunchen) [reply content/zh/examples/admin/sched/my-scheduler.yaml]: Now use  ubuntu18.04.

**Comment by tengqm** [comment]
Thanks for the huge effort.
/lgtm
/approve

**Pull request merged**